rmf_demo_panel/rmf_demo_panel/static/dashboard_config.json

```diff
@@ -0,0 +1,38 @@
+{ 
```

Comment:
I see that there are `dashboard_config.json` files for `office`, `airport` and `clinic` worlds. What is the purpose of this file?

Reply:
following from ^, I assumed that each world would have its own config file, so how it's implemented now is that clicking between tabs loads the different form options for the frontend. I think the outcome is that eventually the `dashboard_config` file would be appended to the `main.json` of each world in `rmf_dashboard_resources`